Symfony/CS/ConfigurationResolver.php

```diff
@@ -84,6 +99,10 @@ public function getFixers()
 
     public function getProgress()
     {
+        if (!method_exists($this->config, 'getHideProgress')) {
```

Comment:
It's more generic indeed, but also more error prone (think of `getHideProgress($requiredArg)`). But yeah, it's not that important.